packages/runtime/src/next/client.ts

```diff
@@ -40,9 +40,25 @@ import {
 } from '../state/react-page'
 import { getMakeswiftSiteVersion, MakeswiftSiteVersion } from './preview-mode'
 
-export type MakeswiftPage = {
+export type MakeswiftPageResult = {
   id: string
   path: string
+  title: string | null
+  description: string | null
+  canonicalUrl: string | null
+  socialImageUrl: string | null
+  sitemapPriority: number | null
+  sitemapFrequency: 'always' | 'hourly' | 'daily' | 'weekly' | 'monthly' | 'yearly' | 'never' | null
+  createdAt: string
+  updatedAt: string
+  publishedAt: string | null
+  isOnline: boolean | null
+  excludedFromSearch: boolean | null
+  locale: string
+  localizedVariants: {
+    locale: string
+    path: string
+  }[]
 }
```

Comment:
How do you guys feel about creating a Zod schema so that we can verify the response at runtime?